js/main.js

```diff
@@ -26,4 +30,64 @@ $form.addEventListener('submit', function (event) {
   data.entries.unshift(entry);
   $placeholder.src = 'images/placeholder-image-square.jpg';
   $form.reset();
+
+  window.location.reload();
```

Comment:
you cannot reload the application during run time to update the DOM. Instead, you want to read through the hints about using the `prepend` method to add the new entry to the DOM when the form is submitted. Make sure to reach out with any questions!